pymeasure/instruments/instrument.py

```diff
@@ -35,8 +35,8 @@
 
 
 class Instrument(object):
-    """ This provides the base class for all Instruments, which is 
-    independent of the particular Adapter used to connect for 
+    """ This provides the base class for all Instruments, which is
```

Comment:
It looks like your editor is automatically removing spaces. I would be careful with this behavior, but it looks like there are no material changes.

Reply:
I think stripping ending white space is one of my own bad habits when looking through files, I'll try to keep from doing that in the future.

Reply:
No worries. It just required more review to ensure the files did not change.